goalie/go_mesh_seq.py

```diff
@@ -150,18 +155,26 @@ def indicate_errors(
                 u_star_e[f] = Function(fs_e)
 
             # Get forms for each equation in enriched space
-            forms = enriched_mesh_seq.form(i, mapping)
-            if not isinstance(forms, dict):
-                raise TypeError(
-                    "The function defined by get_form should return a dictionary"
-                    f", not type '{type(forms)}'."
-                )
+            if not timedep_const:
+                forms = enriched_mesh_seq.form(i, mapping)
+                if not isinstance(forms, dict):
+                    raise TypeError(
+                        "The function defined by get_form should return a dictionary"
+                        f", not type '{type(forms)}'."
+                    )
```

Comment:
After my changes, this check is not being done if `timedep_const=True`. But I thought that we could find a better place for this to live anyway.

Reply:
Surely `forms` needs to be defined in the `timedep_const` case? Did you have an idea of the 'better place' to put this?

Reply:
Yup! See #169 :)